Makefile

```diff
@@ -12,8 +12,6 @@ venv:
 
 setup:
 	@pip install -U -e .[dev]
-	@npm install
```

Comment:
Are we sure we are removing this npm install? Was not this required for pyRight?
@sbimochan can you add comment to this?

Reply:
Yes it is required. Will add it